packages/button/index.tsx

```diff
@@ -22,11 +22,13 @@
 
 import React from 'react';
 import classnames from 'classnames';
-import {withRipple, InjectedProps} from '@material/react-ripple';
+import {withRipple, InjectedProps, RippledComponentProps} from '@material/react-ripple';
```

Comment:
@moog16 I traced this to a typescript error not unlike https://github.com/microsoft/TypeScript/pull/24874 (cc: @weswigham)

If we do not explicitly import `RippledComponentProps` the resulting [typefile](https://termbin.com/w4ez) attempts a relative import at the bottom (`import("../ripple").RippledComponentProps`).

If we do explicitly import `RippledComponentProps` (and use/export it as required to avoid lint errors) the resulting [typefile](https://termbin.com/6ow4) is able to directly use the imported type (`RippledComponentProps`).

This appears to be an issue in typescript resolution/expansion itself. The import at the top is left alone, but the generated one at the bottom will default to a relative import.

Reply:
We can wait for a fix in the typescript repo (I do not have the time to dig into that) or add these dummy type exports to allow proper import of `RippledComponentProps` in all 10 affected packages (see this file for an example).

Reply:
The TS PR that you linked should already be in the TS library, no? 

I think we'll be better off adding the line you added to all the components needing it.